test/stripe/converter_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule Stripe.ConverterTest do
+  use ExUnit.Case
+
+  alias Stripe.Converter
+
+  defmodule Person do
+    defstruct [:email, :first_name, :last_name, :legal_entity, :metadata]
+    def relationships, do: %{}
+  end
+
+  defmodule PersonWithMissingRelationship do
```

Comment:
This is actually an amazing module and should never change. If ever needed, I am willing to constantly rearchitect everything around this, no matter how tedious or unnecessary.